resources/assets/styles/main.scss

```diff
@@ -1,7 +1,3 @@
-@import "~bootstrap/scss/functions";
-@import "common/variables";
```

Comment:
Actually, this line can be kept in the `main.scss` file.
If we remove it, other frameworks variables are not loaded, and `~bootstrap/scss/functions` doesn't seem to override variables.

Reply:
i think the concern is using a bootstrap function in your variables

Reply:
Bootstrap themselves [use functions in variables](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss) so it's confusing if we can't do the same in Sage.

Reply:
You can always take the example from zurb/foundation, calling `functions.scss` inside `variables.scss`:

https://github.com/zurb/foundation-sites/blob/eaa79c89a8c5dc89f8fa1e6916ce5dea6cc55866/scss/settings/_settings.scss#L63

Reply:
You mean [here](https://github.com/roots/sage-installer/blob/master/src/Presets/stubs/Bootstrap/styles/common/_variables.scss) in sage-installer?

That would work. Is that less messy than including `common/variables.scss` in each autoload? Is it more or less flexible in the future?

Reply:
Yes, yes, yes. 😆 👍 
At least, it should be, imo.

Reply:
I'm comfortable with that solution. @retlehs do you have an opinion on including the BS functions right in the variables.scss file?

And forgive my ignorance, should I do that on this PR or is that a sage-installer only thing?

Reply:
sounds good to me, you'll want to do that both on this repo (as it should function properly w/ bootstrap without the installer) and sage-installer